_data/retail.yml

```diff
@@ -153,6 +153,14 @@ websites:
       - software
     doc: https://help.etsy.com/hc/en-us/articles/115015569567
 
+  - name: EVENTIM
```

Comment:
```suggestion
  - name: Eventim (DE)
```

Reply:
They use the all-capitals notation throughout, with the exception of the logo, where it's all lowercase. :man_shrugging:

I'm okay with the country marker, though. How about `EVENTIM (DE)`? Or is that too loud?

Reply:
On twitter and facebook they use lowercase.
http://facebook.com/eventimDE
https://twitter.com/eventimDE
Also, the title on their page is `Eventim: [...]`

Reply:
Neither the facebook nor the twitter handle are the official company name, and the page title seems to be a mistake.

Although it may seem ugly, the all-capital spelling is the correct one (for Germany). Many of the [international branches](https://www.eventim.de/campaign/international/) use it as well.

Reply:
I checked whether it was possible to have your name begin with a lowercase letter on facebook (I know twitter has no problems with that) and it is, see f/edX. 
We've used the imprint of sites before to determine their legal name, which in this case is
CTS EVENTIM AG & Co. KGaA (company) and EVENTIM Management AG (shareholder).
Having the logo in all lowercase letters where the company name is all capitals might be the outright the stupidest marketing decision one could take but here we are. 
I'd, therefore, root for `EVENTIM`. I could also perfectly live with `eventim` that would be consistent with the logo. But I don't think `Eventim` "looks" right.